packages/xpetra/src/BlockedCrsMatrix/Xpetra_MapExtractor_decl.hpp

```diff
@@ -115,7 +119,7 @@ namespace Xpetra {
      *
      * \param map BlockedMap defining the block structure of the multi vector
      */
-    MapExtractor(const Teuchos::RCP< const BlockedMap > &map);
+    MapExtractor(const Teuchos::RCP< const Xpetra::BlockedMap<LocalOrdinal,GlobalOrdinal,Node> > &map);
```

Comment:
You don't need to leave a space between the two closing angle brackets any more (with C++11).

Reply:
@mhoemmen I know. In some places I'm changing things to the C++11 style but there's a lot of code stuff in these ETI functions that I'm not really touching so it's just staying how it was in the original, pre-ETI code.